src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -1589,6 +1589,54 @@ public partial class View
             return view.backgroundExtraData == null ? VisualTransformPolicyType.Absolute : view.backgroundExtraData.CornerRadiusPolicy;
         });
 
+        /// <summary>
+        /// BorderlineWidth Property
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty BorderlineWidthProperty = BindableProperty.Create(nameof(BorderlineWidth), typeof(float), typeof(View), default(float), propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var view = (View)bindable;
+            (view.backgroundExtraData ?? (view.backgroundExtraData = new BackgroundExtraData())).BorderlineWidth = (float)newValue;
+            view.ApplyBorderline();
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var view = (View)bindable;
+            return view.backgroundExtraData == null ? 0.0f : view.backgroundExtraData.BorderlineWidth;
+        });
+
+        /// <summary>
+        /// BorderlineColor Property
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty BorderlineColorProperty = BindableProperty.Create(nameof(BorderlineColor), typeof(Color), typeof(View), null, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var view = (View)bindable;
+            (view.backgroundExtraData ?? (view.backgroundExtraData = new BackgroundExtraData())).BorderlineColor = (Color)newValue;
+            view.ApplyBorderline();
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var view = (View)bindable;
+            return view.backgroundExtraData == null ? Color.Black : view.backgroundExtraData.BorderlineColor;
```

Comment:
it should be null, 
because, backgroundExtraData.BorderlineColor default value is null
``` c#
internal Color BorderlineColor { get; set; }
```

example....
``` c#
 View view1 = ....
 // Do not set any BackgroundExtraData
 Debug.Assert(view1.BorderlineColor == Color.Black); // according to definition of BorderlineColorProperty

View view2 = ......
 // Set BorderlineOffset
 view2.BorderlineOffset = 0.1; // it will create BackgroundExtraData and 

// according to definition of BorderlineColorProperty, default value is Color.Black
// But actually it is null
 Debug.Assert(view2.BorderlineColor == Color.????); 
 
```

Reply:
fix done. now value will return Color.Black as default.

Reply:
@hinohie Could you check it?
I think, default value of BorderlineColor should be null because default value of `BackgroundExtraData`.`BorderlineColor` is null
``` c#
return view?.backgroundExtraData.BorderlineColor ?? null;
```

Reply:
@myroot  Yes I check it. Now `BackgroundExtraData.BorderlineColor` default value is `Color.Black` 

I don't set this value as null. In DALi area, It'll work as `Color.BLACK` when we don't setup `BorderlineColor`.
When we set default value as `null`, then user may get confuse. 
"Now I look some black line now. but why `BorderlineColor` doesn't return `Color.Black`??"